### ISSUE TITLE
[🐛 Bug]: The 'cache' field on 'RequestInitializerDict' is not implemented.

### ISSUE THREAD

**Issue by Tkko** [cloudflare/next-on-pages#719]
### next-on-pages environment related information

System:
        Platform: darwin
        Arch: arm64
        Version: Darwin Kernel Version 23.4.0: Wed Feb 21 21:44:43 PST 2024; root:xnu-10063.101.15~2/RELEASE_ARM64_T6000
        CPU: (8) arm64 Apple M1 Pro
        Memory: 16 GB
        Shell: /bin/zsh
Package Manager Used: npm (10.5.0)

Relevant Packages:
        @cloudflare/next-on-pages: 1.10.0
        vercel: N/A
        next: N/A

### Description

I deployed Next.js 14 [E-commerce](https://github.com/vercel/commerce/) app on pages and everything is working fine most of the time but I occasionally get this error:
```
{
  cause: 'unknown',
  status: 500,
  message: "The 'cache' field on 'RequestInitializerDict' is not implemented.",
  query: 'something'
  }
```

### Reproduction

- Go to product page (Or any page that uses `cache: 'no-store'` property)
- Refresh the page for couple of time
- The error happens

### Pages Deployment Method

Pages CI (GitHub/GitLab integration)

### Would you like to help?

- [ ] Would you like to help fixing this bug?

**Comment by IgorMinar** [comment]
Thanks for the report. I believe this is a runtime issue: https://github.com/cloudflare/workerd/issues/698

I'll escalate it with the runtime team.

**Comment by tstpierre** [comment]
Obviously, this is a `not for production` bit, thanks to James on the cloudflare discord - I'm able to continue with CI/CD regression testing by modifying the build settings in cloudflare's dashboard to use this canary:

Build command:
`npx @cloudflare/next-on-pages@0.0.0-317e872`

It did at least unblock development.

**Comment by dario-piotrowicz** [comment]
Hello 🙂 Thank you for the issue and apologies for the late reply.

Earlier this year we released our [Cloudflare adapter for OpenNext](https://opennext.js.org/cloudflare), and we have been continuously investing effort in its improvement since then. Existing deployments can continue to use the next-on-pages tooling, but the Cloudflare adapter for OpenNext is currently stable and recommended for deployment of Next.js applications to Cloudflare.

As such, we will no longer be maintaining next-on-pages. We recently deprecated the package on npm and we plan to archive this repo on Monday Sep 29, 2025. In preparation for archival, we are closing all open issues and PRs.

If you have concerns, please feel free to reply here before the repo is archived (after that, it will no longer be possible to comment) or to reach out to our team on [Discord](https://discord.com/invite/cloudflaredev).

Thanks so much for using and contributing to next-on-pages 🧡
